binding/HarfBuzzSharp.Shared/Buffer.cs

```diff
@@ -235,19 +235,21 @@ public void AddCodepoints (IntPtr text, int textLength, int itemOffset, int item
 			if (ContentType == ContentType.Glyphs)
 				throw new InvalidOperationException ("ContentType must not be of type Glyphs");
 
-			HarfBuzzApi.hb_buffer_add_codepoints (Handle, text, textLength, itemOffset, itemLength);
+			HarfBuzzApi.hb_buffer_add_codepoints (Handle, (uint*)text, textLength, (uint)itemOffset, itemLength);
 		}
 
 		public unsafe ReadOnlySpan<GlyphInfo> GetGlyphInfoSpan ()
 		{
-			var infoPtrs = HarfBuzzApi.hb_buffer_get_glyph_infos (Handle, out var length);
-			return new ReadOnlySpan<GlyphInfo> (infoPtrs, length);
+			uint length;
+			var infoPtrs = HarfBuzzApi.hb_buffer_get_glyph_infos (Handle, &length);
+			return new ReadOnlySpan<GlyphInfo> (infoPtrs, (int)length);
 		}
 
 		public unsafe ReadOnlySpan<GlyphPosition> GetGlyphPositionSpan ()
```

Comment:
Hopefully we can change the to a writable Span<T> soon

Reply:
I think we can, but maybe not in this PR. I see 2.7.2 is out, so we can break with 2.7.x.